packages/node/src/indexer/dictionary/v2/ethDictionaryV2.ts

```diff
@@ -192,7 +194,7 @@ export function buildDictionaryV2QueryEntry(
             filter.to !== undefined ||
             filter.function !== undefined
           ) {
-            dictionaryConditions.transactions.push(
+            dictionaryConditions.transactions?.push(
```

Comment:
Should `dictionaryConditions.transactions` be defined if not?

Same applies to logs further down.